src/utils/__tests__/internalLinks-test.js

```diff
@@ -139,6 +139,7 @@ describe('getNarrowFromLink', () => {
       new URL('https://example.com'),
       usersById,
       new Map(streams.map(s => [s.stream_id, s])),
+      userA.user_id,
```

Comment:
This parameter is called `ownUserId`; would this be a good place to use `eg.selfUser`?